src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -1,41 +1,31 @@
 import { Project } from "@/utils/interfaces";
-import {
-  Thumb,
-  Wrapper,
-  InfoWrapper,
-  TextWrapper,
-  Title,
-  Description,
-  Link,
-  ThumbContainer,
-} from "./ProjectCard.styles";
 import { LinkButton } from "../LinkButton/LinkButton";
 
 const ProjectCard = ({ project }: { project: { fields: Project } }) => {
   const { name, description, link, thumb } = project.fields;
 
   return (
-    <Wrapper>
-      <Link href={link}>
-        <ThumbContainer>
-          <Thumb
+    <div className="flex flex-col md:flex-row gap-4 justify-between items-center w-full max-w-full md:max-w-[80%] lg:max-w-[48%] xl:max-w-[45%] mx-auto">
+      <a href={link} className="w-full">
+        <div className="relative w-full overflow-hidden rounded">
+          <img
```

Comment:
Instead of using `<img>` tag use the next's Image tag, then we can set the image's quality